dist/docker-images/ziti-controller/compose.yml

```diff
@@ -0,0 +1,50 @@
+
+volumes:
+  ziti-controller:
+    driver: local
+
+services:
+  chown-volume:
+    image: busybox
+    command: chown -R ${ZIGGY_UID:-2171} /mnt
+    volumes:
+      - ziti-controller:/mnt
+
+  ziti-controller:
+    image: ${ZITI_CONTROLLER_IMAGE:-openziti/ziti-controller}
+    depends_on:
+      chown-volume:
+        condition: service_completed_successfully
+    volumes:
+      - ziti-controller:/mnt
+    working_dir: /mnt
+    # these declared vars pass through to container and should be assigned in an .env file or exported from parent env
+    # to ensure consistency throughout the compose project
+    environment:
+      # *** these are the important vars to set ***
+      ZITI_CTRL_ADVERTISED_ADDRESS:                                   # domain name of this controller (required)
+      ZITI_CTRL_ADVERTISED_PORT: ${ZITI_CTRL_ADVERTISED_PORT:-1280}   # exposed port of this controller
+      ZITI_PWD: ${ZITI_PWD:-}                                         # password for the default admin user
```

Comment:
i think `:-` should be removed here and make the user set it.

Reply:
It's optional in this revision because it's only required for the bootstrapping (first) run. I'll split the inputs between two containers: bootstrap and run. That way, it will be clearer which variables apply to which.